sklearn/linear_model/tests/test_coordinate_descent.py

```diff
@@ -24,9 +28,11 @@
 from sklearn.utils._testing import TempMemmap
 from sklearn.utils.fixes import parse_version
 
-from sklearn.linear_model import Lasso, \
+from sklearn.linear_model import Lasso, LassoLars, Lars, LinearRegression, \
```

Comment:
Let's make an import per line

```python
from sklearn.linear_model import Lasso
from sklearn.linear_model import LassoLars
...
```

It will avoid diff in the future